scripts/handlers/createPost.mjs

```diff
@@ -0,0 +1,24 @@
+import { createPost } from "../api/posts/index.mjs";
+
+export function setCreatePostFormListener() {
+  const form = document.querySelector("#createPost");
+
+  if (form) {
+    form.addEventListener("submit", async (event) => {
+      event.preventDefault();
+
+      const formData = new FormData(form);
+      const post = Object.fromEntries(formData.entries());
+
+      post.tags = post.tags.split(",").map(tag => tag.trim());
```

Comment:
You should add a similar check to when you're updating a post, to when creating one.

With your current solution, the `tags` array will end up sending an array with an empty string rather than just an empty array. Since there's no check for this when displaying posts, each post created with no tags, will display an empty hashtag `#` on it.

There's two ways you can do this:
- Set the array to be empty, similarly to how it's done in updatePost.
- Delete the `tags` property completely, and not send it 

Both will have the same effect, but one will send an empty array while the other deletes it completely and relies on the API to set the default.
```suggestion
      if (!post.tags) {
        post.tags = []; // or: delete post.tags;
      } else {
        post.tags = post.tags.split(",").map(tag => tag.trim());
      }
```